### ISSUE TITLE
Remove method sets _loop to false but it is set back to true somehow

### ISSUE THREAD

**Issue by sepans** [processing/p5.js#502]
When p5 remove method is being called (in my case when there is an error in setup/draw functions) _loop is set to false as expected and the intervals are cleared, but mysteriously it is set back to true so it continues setting the interval in the _draw method and calling userDraw (and flooding console with errors). I am sure the loop method is not being called and I couldn't find anywhere else that loop is set. 

Code [here](https://github.com/sepans/p5-element/blob/master/p5-element.html#L121)

Thanks

**Comment by lmccart** [comment]
thanks, I'll look into this!

**Comment by lmccart** [comment]
hey @sepans hmm it's a little difficult for me to track down the issue in the code you linked.
this example works for me:

```javascript
var myp5 = new p5(function( sketch ) {

  sketch.setup = function() {
    sketch.createCanvas(400, 400);
  };

  sketch.draw = function() {
    sketch.background(0);
    sketch.rect(sketch.width/2, sketch.height/2, 50, 50);
  }

  sketch.mousePressed = function() {
    sketch.remove();
  }

});
```

could you make a simpler example that illustrates the issue?

**Comment by sepans** [comment]
Thanks @lmccart . Will do that. One thing I noticed is that when there is an error in userDraw function in _draw method two instance of p5 run in parallel. Most likely my code. I will investigate that.

**Comment by workergnome** [comment]
Closing this as **Cannot Reproduce**.  @sepans, if you can duplicate this issue, please feel free to reopen or create a new issue!

Thanks, all.
